### DISCUSSION TITLE
Otimização na criação dos registros na base de dados.

### DISCUSSION THREAD

**Discussion by LeandroVCastro** [raulfdm/taco-api#64] [Ideas]
Olá. Tudo bem?

Tenho utilizado essa API para poder fazer umas tarefas locais.

Olhando a parte de popular o banco de dados, percebo uma oportunidade de usar o método createMany do Prisma em vez do create.

Isso permitiria a inserção mais rápida dos registros na base de dados.
Referência: https://www.prisma.io/docs/concepts/components/prisma-client/crud#create-multiple-records

O que você acha dessa alteração?
Caso aprove, eu mesmo posso fazer um PR e subir aqui pra você poder revisar.

Abraços e obrigado pelo repositório.

**Comment by raulfdm** [comment]
Fala Leandro, beleza?

Manda bala. Abre o PR que eu testo local e faço uma release com a alteração.

Pra ficar um pouco mais cientifico, consegue mensurar quão mais rápido? Traz uns dados pra gente! 😄 

Obrigado pela contribuição 🙌

**Comment by LeandroVCastro** [comment]
Obrigado pela resposta.
Ainda não sei os números exatos.
Vou fazer uns testes nos próximos dias e vou juntar com o PR.
